Index: storage/framework/views/3bc419d60a4b780ff092babbd2b4069e533fb049.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/storage/framework/views/3bc419d60a4b780ff092babbd2b4069e533fb049.php b/storage/framework/views/3bc419d60a4b780ff092babbd2b4069e533fb049.php
--- a/storage/framework/views/3bc419d60a4b780ff092babbd2b4069e533fb049.php	(date 1616653762497)
+++ b/storage/framework/views/3bc419d60a4b780ff092babbd2b4069e533fb049.php	(date 1616653762497)
@@ -83,15 +83,16 @@
                             <button class="close" data-dismiss="modal">&times;</button>
                         </div>
                         <div class="modal-body">
-                            <form action="#" method = "post" enctype="multipart/form-data">
+                            <form action="#" method = "POST" enctype="multipart/form-data">
                                 <?php echo csrf_field(); ?>
                                 <div class="form-group">
                                     <input name="title" class="form-control" type="text" placeholder="Enter Post Title">
                                 </div>
                                 <div class="form-group">
                                     <label style="font-size:80px;cursor:pointer;" for="fimg"><i class="fas fa-file-image"></i></label>
-                                    <input style="display:none;" name="fimg" type="file" id="fimg">
-                                    <img id="featured_image_load" src="" >
+                                    <br>
+                                    <input style="display:none;" name="fimage" type="file" id="fimg">
+                                    <img style="max-width: 100%;" id="featured_image_load" src="" >
                                 </div>
                                 <div class="form-group">
                                     <input class="btn-primary btn-sm btn" type="submit" value="Add">
Index: storage/framework/views/cb50e4f7404fe6113c7e2b4154ffc2f42d337580.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/storage/framework/views/cb50e4f7404fe6113c7e2b4154ffc2f42d337580.php b/storage/framework/views/cb50e4f7404fe6113c7e2b4154ffc2f42d337580.php
--- a/storage/framework/views/cb50e4f7404fe6113c7e2b4154ffc2f42d337580.php	(date 1616653762493)
+++ b/storage/framework/views/cb50e4f7404fe6113c7e2b4154ffc2f42d337580.php	(date 1616653762493)
@@ -68,7 +68,7 @@
             //Post Featured Image change
             $(document).on('change',"input#fimg",function(event){
                 event.preventDefault();
-                let post_image_url = URl.createObjectURL(event.target.files[0]);
+                let post_image_url = URI.createObjectURL(event.target.files[0]);
                 $('img#featured_image_load').attr('src', post_image_url);
 
 
Index: resources/views/admin/post/index.blade.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/resources/views/admin/post/index.blade.php b/resources/views/admin/post/index.blade.php
--- a/resources/views/admin/post/index.blade.php	(date 1616653762489)
+++ b/resources/views/admin/post/index.blade.php	(date 1616653762489)
@@ -85,15 +85,16 @@
                             <button class="close" data-dismiss="modal">&times;</button>
                         </div>
                         <div class="modal-body">
-                            <form action="#" method = "post" enctype="multipart/form-data">
+                            <form action="#" method = "POST" enctype="multipart/form-data">
                                 @csrf
                                 <div class="form-group">
                                     <input name="title" class="form-control" type="text" placeholder="Enter Post Title">
                                 </div>
                                 <div class="form-group">
                                     <label style="font-size:80px;cursor:pointer;" for="fimg"><i class="fas fa-file-image"></i></label>
+                                    <br>
                                     <input style="display:none;" name="fimage" type="file" id="fimg">
-                                    <img id="featured_image_load" src="" >
+                                    <img style="max-width: 100%;" id="featured_image_load" src="" >
                                 </div>
                                 <div class="form-group">
                                     <input class="btn-primary btn-sm btn" type="submit" value="Add">
Index: resources/views/layouts/partials/script.blade.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/resources/views/layouts/partials/script.blade.php b/resources/views/layouts/partials/script.blade.php
--- a/resources/views/layouts/partials/script.blade.php	(date 1616653762484)
+++ b/resources/views/layouts/partials/script.blade.php	(date 1616653762484)
@@ -68,7 +68,7 @@
             //Post Featured Image change
             $(document).on('change',"input#fimg",function(event){
                 event.preventDefault();
-                let post_image_url = URl.createObjectURL(event.target.files[0]);
+                let post_image_url = URI.createObjectURL(event.target.files[0]);
                 $('img#featured_image_load').attr('src', post_image_url);
 
 
